src/servers/graphql-server.ts

```diff
@@ -215,6 +216,56 @@ export const startApolloServer = async ({
   const app = express()
   const httpServer = createServer(app)
 
+  const partialAccountTransactionsPlugin: ApolloServerPlugin = {
+    requestDidStart: async () => {
+      const normalizeWalletTransaction = (edge: { node: WalletTransaction }) => ({
+        ...edge,
+        node: {
+          ...edge.node,
+          settlementDisplayPrice: {
+            ...edge.node.settlementDisplayPrice,
+            base: Number(edge.node.settlementDisplayPrice.base),
+            offset: Number(edge.node.settlementDisplayPrice.offset),
+          },
+        },
+      })
```

Comment:
this hide query behavior, if we can move this to a "common" place for txs would be better

Reply:
also.. is really an user facing error if we cant get the pending onchain txs?

Reply:
> this hide query behavior, if we can move this to a "common" place for txs would be better

Like this you mean? https://github.com/GaloyMoney/galoy/pull/2651/commits/16a739127cd8bcec3e45c7ab774f2078515e8967

Reply:
> also.. is really an user facing error if we cant get the pending onchain txs?

If it's a partial error then ideally it doesn't reach the user, but maybe it's useful for the frontend to know via the API still?